frontend/src/app/organization/organization-details.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/switchMap';
+
+import { Organization } from './organization.model';
+import { OrganizationService } from './organization.service';
+import { environment } from '../../environments/environment';
+import { AuthService } from '../auth.service';
+import { User } from '../user';
+
+@Component({
+  selector: 'volontulo-organization-details',
+  templateUrl: './organization-details.component.html',
+  styleUrls: ['./organization-details.component.css'],
+})
+export class OrganizationDetailsComponent implements OnInit {
+  currentUser: User;
+  organization$: Observable<Organization>;
+  djangoRoot: string = environment.djangoRoot;
+  isUserOrgMember$: Observable<boolean>;
+
+  constructor(
+    private activatedRoute: ActivatedRoute,
+    private organizationService: OrganizationService,
+    private authService: AuthService,
+  ) {}
+
+  ngOnInit() {
+    this.authService.changeUserEvent
+      .subscribe(user => { this.currentUser = user; }
```

Comment:
I've just taken a look once more at this PR and I'm afraid it will not work correctly when you log out while visiting this view. You'll set the current user, but it will be ignored in your view where you check whether the user is an org member.

To acknowledge the fact the `currentUser` might also change in time, you can use `combineLatest` from rxjs to combine latest organization with latest user and then `map` to have your boolean.

BTW, this `changeUserEvent` should not really be an event, but rather a user should be come from `BehaviourSubject` in `authService`. I think you can ignore that for now until it's refactored in all the necessary places.

You can try to utilize this `combineLatest` on your own, if you get stuck, let us know.